### PR TITLE
+11 Icons, +1 link

### PR DESCRIPTION
**Pull request by airon-main**
# Description

The pr before has a lot of file changed so idk how to fix that. i'll just remake the pr (it works somehow)
if there's a way to fix it please tell me. i'm still learning

### Added
TheFor (`xyz.thefor.habits.habits`)
Home Workout (`homeworkout.homeworkouts.noequipment`)
Pako Highway (`com.treemengames.pakohighway`)
Godot Editor 4 (`org.godotengine.editor.v4`)
Recharge RC (`com.HonduneGames.ReCharge`)
Polytopia (`air.com.midjiwan.polytopia`)
CarX Street (`com.carxtech.sr`)
Rebel Racing (`com.hutchgames.rebelracing`)
System UI Tuner (`com.bryancandi.android.uituner`)
R.A.C.E. (`com.smokoko.race`)
Power Marathon (`com.transsion.batterylab.icon`)

### Linked
Ruler (`org.nixgame.ruler` → `ruler`)

## Contributor's checklist

- [x] I made the icons following [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: svgs/power_marathon_icon.svg
Comment (x9136) [inline svgs/power_marathon_icon.svg:1]: There is a feeling that this app is now called BatteryLab. I hope you found and installed the apk to make sure that the app exists and is called Power Marathon.

In any case, all the icons that I found are far from your option. First of all, I suggest confirming the existence of the app through installation on the phone.

File: svgs/recharge_rc.svg
Comment (x9136) [inline svgs/recharge_rc.svg:1]: It is worth moving 1-2 px up.

File: svgs/system_ui_tuner.svg
Comment (x9136) [inline svgs/system_ui_tuner.svg:1]: Screenshot 2024-03-08 at 11 04 08

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/power_marathon_icon.svg
Comment (airon-main) [reply svgs/power_marathon_icon.svg:1]: It does exist

File: svgs/recharge_rc.svg
Comment (airon-main) [reply svgs/recharge_rc.svg:1]: Aight

File: app/assets/appfilter.xml
[the remark this answers is not in the document]
Comment (airon-main) [reply app/assets/appfilter.xml:1]: Got it

File: svgs/carx.svg
[the remark this answers is not in the document]
Comment (airon-main) [reply svgs/carx.svg:1]: Sure

File: svgs/godot.svg
[the remark this answers is not in the document]
Comment (airon-main) [reply svgs/godot.svg:1]: Noted

File: svgs/system_ui_tuner.svg
Comment (airon-main) [reply svgs/system_ui_tuner.svg:1]: Yea. I usually forgot about that

File: app/assets/appfilter.xml
[the remark this answers is not in the document]
Comment (airon-main) [reply app/assets/appfilter.xml:1]: image

[the remark this answers is not in the document]
Comment (x9136) [reply app/assets/appfilter.xml:1]: As far as I can see, you need an explanation regarding the issue with appfilter.xml.

I do not know why you did not pay attention to the screenshot in the first comment, but even so, I cannot merge the PR until you are sure that Lawnicons builds correctly.

If the info in the screenshot does not help to figure out the issue, then the point is simple: the names of the icons should be the same as the drawables.

[the remark this answers is not in the document]
Comment (airon-main) [reply app/assets/appfilter.xml:1]: oh...

the color is gray for the first screenshot

yea sorry about that

File: svgs/pako_highway.svg
[the remark this answers is not in the document]
Comment (airon-main) [reply svgs/pako_highway.svg:1]: figma export do get a bit quirky

File: app/assets/appfilter.xml
[the remark this answers is not in the document]
Comment (airon-main) [reply app/assets/appfilter.xml:1]: > Screenshot 2024-03-08 at 10 16 22

image

[the remark this answers is not in the document]
Comment (airon-main) [reply app/assets/appfilter.xml:1]: it built fine in my device

[the remark this answers is not in the document]
Comment (airon-main) [reply app/assets/appfilter.xml:1]: image

**Pull request merged**